idkit-kotlin/src/main/java/com/worldcoin/idkit_kotlin/Types.kt

```diff
@@ -229,6 +246,12 @@ enum class CredentialCategory {
      * The set of NFC credentials with active or passive authentication.
      */
     @SerialName("secure_document") SECURE_DOCUMENT,
+
+    /**
+     * The set of Orb-derived credentials (I.E Iris Code)
+     */
+    @SerialName("orb")
+    ORB,
```

Comment:
is the category orb? or should the category be personhood?

don't have a preference, curious how we're curating these categories

Reply:
I think I'm convinced that `personhood` is a good name for this credential category. Let's go with that. I'm about to merge the `idkit-swift` changes with it.

Reply:
https://github.com/worldcoin/idkit-swift/pull/18